playbooks/_defaults.yml

```diff
@@ -0,0 +1,4 @@
+# This file can be used to override default
+# role values. This is useful for "downstream"
+# packaging and opinionated releases.
+---
```

Comment:
is not clear to me why we need both build/_defaults.yml and playbooks/_defaults.yml.

Reply:
playbooks/_defaults.yml is empty by default and is there just to make sure the vars_files will not fail. The build one is the one supposed to replace the empty playbooks/ one with actual content.

Reply:
Can you add at least as a comment what could be the variables with examples maybe ?